app/src/main/java/org/koitharu/kotatsu/base/ui/BaseActivity.kt

```diff
@@ -34,6 +34,7 @@ abstract class BaseActivity<B : ViewBinding> : AppCompatActivity(), OnApplyWindo
 	private var lastInsets: Insets = Insets.NONE
 
 	override fun onCreate(savedInstanceState: Bundle?) {
+		setTheme(R.style.AppTheme)
```

Comment:
I think this should be in `MainActivity` only, because

1. Only `MainActivity` have `AppTheme.Splash` in manifest
2. Other activities can have other themes, not only `AppTheme` (maybe in future)